tests/integration/tests/menu.js

```diff
@@ -62,21 +62,14 @@ test('menu open favorite', async () => {
 });
 
 test('one panel open at a time', async () => {
-  expect.assertions(2);
+  expect.assertions(1);
   await page.goto(APP_URL);
-  const servicePanelOpen = await page.waitForSelector('.service_panel--active');
```

Comment:
The `service_panel` isn't "active" (`display:block`) or "inactive" (`display:none`) anymore. It either is in the DOM or not, so we have to adapt the tests.